cycode/cyclient/models.py

```diff
@@ -171,6 +171,23 @@ def __init__(
         self.err = err
 
 
+class ScanReportUrlResponse(Schema):
```

Comment:
as i understand this is DTO class and should not be inhered from Schema. you already have schema below